ERCS/erc-5521.md

```diff
@@ -1,10 +1,10 @@
 ---
-eip: 5521
+erc: 5521
```

Comment:
```suggestion
eip: 5521
```

Thanks for noticing this. We aren't yet ready to make this conversion. It is on the todo list in #1.